models/biocro/R/run.biocro.R

```diff
@@ -51,111 +44,177 @@ run.biocro <- function(lat, lon, metpath, soil.nc = NULL, config = config, coppi
       day1 <- NULL
       dayn <- NULL
     }
+    if (!is.null(day1) && day1 > lubridate::yday(starti)) {
+      lubridate::yday(starti) <- day1
+    }
+    if (!is.null(dayn) && dayn < lubridate::yday(starti)) {
+      lubridate::yday(plant_date) <- dayn
```

Comment:
What is `plant_date`? I would think it means the date the crop was planted, but it's being assigned the value of `dayn`, which is `config$simulationPeriod$dateofharvest` at this point.

Reply:
Also, the conditional is a check against `starti`. If this is right, it should be commented to explain it.

Reply:
Good catch -- `plant_date` isn't a thing and that line should have been `endi`.

Reply:
Fixed in 1ee5c70fb423491dd559603f0da75d0df4ba5b68